components/calendar/calendar.js

```diff
@@ -199,32 +208,41 @@ class Calendar extends LocalizeStaticMixin(LitElement) {
 				justify-content: center;
 				margin-left: auto;
 				margin-right: auto;
+				opacity: 0;
 				padding: 4px;
 				position: relative;
 				text-align: center;
 				width: calc(2rem - 4px);
 			}
 
-			:host(.d2l-calendar-next) .d2l-calendar-title .d2l-heading-4,
-			:host(.d2l-calendar-prev) .d2l-calendar-title .d2l-heading-4,
-			:host(.d2l-calendar-initial-month) .d2l-calendar-title .d2l-heading-4,
-			:host(.d2l-calendar-next) .d2l-calendar-date,
-			:host(.d2l-calendar-prev) .d2l-calendar-date,
-			:host(.d2l-calendar-initial-month) .d2l-calendar-date {
+			.d2l-calendar-next .d2l-calendar-date div {
+				left: 10px;
+			}
+
+			.d2l-calendar-prev .d2l-calendar-date div {
+				left: -10px;
+			}
+
+			.d2l-calendar-next .d2l-calendar-title .d2l-heading-4,
+			.d2l-calendar-prev .d2l-calendar-title .d2l-heading-4,
+			.d2l-calendar-initial-month .d2l-calendar-title .d2l-heading-4,
+			.d2l-calendar-next .d2l-calendar-date div,
+			.d2l-calendar-prev .d2l-calendar-date div,
+			.d2l-calendar-initial-month .d2l-calendar-date div {
```

Comment:
Could use like an additional class like `d2l-calendar-animating` or something that's applied whenever any of them are happening -- simplify the CSS.